app/src/components/Form/DemoValidationFormFlow.js

```diff
@@ -0,0 +1,123 @@
+
```

Comment:
Delete this file if we don't need it.

Reply:
I think we need this, they will be a lot of validation form check in front end part which is useful and don't even wait for backend to retrieve the error. It's like pre-validation check. I will save this in my local or upload this file for them before I delete it.

Reply:
Removed it, backup in Slack @general channel